docs/user_guide/file_transfer_dotnet/README.md

```diff
@@ -1,5 +1,7 @@
 # FirecREST File Transfer with C# .NET
 ## General description
+The .NET solution file (`data_transfer_dotnet.sln`) is available by cloning this repository and accessing to the pat [docs/user_guide/file_transfer_dotnet](.).
```

Comment:
Could we add the actual commands you need to do, something like

```
git clone http....
cd docs/user_guide/file_transfer_dotnet
```

Just to add more visual aid to the users